.github/CODEOWNERS

```diff
@@ -48,11 +52,14 @@
 
 
 # Dev workflow team.
+/core/templates/dev/head/css/.stylelintrc @apb7
 /scripts/ @apb7
 
 
 # Exploration project.
 /core/controllers/editor*.py @DubeySandeep
+/core/controllers/reader*.py @aks681
+/core/controllers/resources*.py @seanlip
```

Comment:
@vojtechjelinek can take this one, I think.

Reply:
Yeah, sure.

Reply:
Done!